.buildkite/pipeline.yml

```diff
@@ -787,6 +788,24 @@ steps:
         agents:
           slurm_mem: 20GB
 
+  - group: "CUDA"
+    steps:
+
+      - label: "CUDA: CUDA implicit baro wave"
```

Comment:
I'm noticing that we labeled other GPU jobs with the `GPU` label rather than `CUDA`. I think we should be consistent.

Reply:
sure, we can change the name